pcsx2-qt/MainWindow.cpp

```diff
@@ -2867,11 +2867,16 @@ void MainWindow::doDiscChange(CDVD_SourceType source, const QString& path)
 	bool reset_system = false;
 	if (!m_was_disc_change_request)
 	{
-		const int choice = QMessageBox::question(this, tr("Confirm Disc Change"),
-			tr("Do you want to swap discs or boot the new image (via system reset)?"), tr("Swap Disc"), tr("Reset"), tr("Cancel"), 0, 2);
-		if (choice == 2)
-			return;
-		reset_system = (choice != 0);
+		QMessageBox messageBox(QMessageBox::Question, tr("Confirm Disc Change"),
+			tr("Do you want to swap discs or boot the new image (via system reset)?"));
+		messageBox.addButton(tr("Swap Disc"), QMessageBox::ActionRole);
+		QPushButton *resetButton = messageBox.addButton(tr("Reset"), QMessageBox::ActionRole);
+		QPushButton *cancelButton = messageBox.addButton(QMessageBox::Cancel);
+		messageBox.setDefaultButton(cancelButton);
+		messageBox.exec();
+
+		if (messageBox.clickedButton() == cancelButton) return;
```

Comment:
`return` on newline (or just clang-format the function).

Reply:
Yeah, my normal code formatting was creeping in, and I've had times before where clang format has changed formatting throughout the file I was working on when I only wanted to correct the formatting I was working on. Since this is a new file, that should be safe, though...

Reply:
Yeah, I usually format the whole file for everything in the Qt subdirectory before committing.